src/main/java/org/example/Main.java

```diff
@@ -20,5 +20,10 @@ public static void main(String[] args) {
         stringexample str = new stringexample();
         String message = "Hi, today is a very nice day.";
         str.printMessageDetails(message);
-    }
+        ByteValidator bv = new ByteValidator();
+        byte[] byteArray ="Hello".getBytes();
+        String validCharacters = "Hello";
+        ByteValidator bv1 = bv;
+        bv1.isValidByteArray(byteArray, validCharacters);
```

Comment:
We don't want to test our code in a main method. We should use unit test class only

Reply:
I wrote this here not to test it in main but to see how it works in main.